schemas/common_definitions.json

```diff
@@ -63,11 +56,30 @@
           "items": {
             "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders"
           }
+        },
+        "definition": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/definition"
         }
       },
       "additionalProperties": false
     }
   },
+  "definition": {
+    "description": "Definition title and contents.",
+    "items": {
+      "type": "object",
+      "required": ["title", "contents"],
+      "properties": {
+        "title": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+        },
+        "contents": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/contents"
+        }
+      }
+    },
+    "additionalProperties": false
```

Comment:
This is 1 level to deep, it should be directly before or after "properties" object.

Reply:
I worked that out when my runner PR went bang